Lib/fontTools/subset/__init__.py

```diff
@@ -2902,7 +2902,7 @@ def main(args=None):
             text += g[7:]
             continue
         if g.startswith('--text-file='):
-            text += open(g[12:]).read().replace('\n', '')
+            text += open(g[12:], encoding='utf-8').read().replace('\n', '')
```

Comment:
Ah, I forgot that we have a [backport of python3 open()](https://github.com/behdad/fonttools/blob/master/Lib/fontTools/misc/py23.py#L178) in fonttools.